hca_bundle_tools/file_metadata_to_csv.py

```diff
@@ -48,8 +48,9 @@ def __init__(self, order = None, ignore = None, format_filter = None):
             "file_name"
         ]
 
-        self.default_format_filter = format_filter if format_filter else ["fastq.gz"]
-
+        self.default_format_filter = format_filter
```

Comment:
I think we just need one mechanism to filter files, this should either be files we include or files to ignore, but not both

Reply:
Most Unix tools (tar, find, rsync, …) have includes and excludes. A reasonably useful filter implementation needs both. Then again, we shouldn't worry about the CLI too much. The command line usage of this script will go away soon and it will be absorbed into 1) the DCP-CLI (as a post-processing step of the manifest download and 2) the Azul indexer keeping the per-project TSVs up-to-date automatically.

Reply:
OK, I won't argue with Unix then :).